src/views/manage/ManageForm.jsx

```diff
@@ -1,65 +1,156 @@
-import React from 'react';
-import { Heading } from '@datapunt/asc-ui';
-import { Formik } from 'formik';
-import { initalValues } from './FormFields';
+import React, { useEffect } from 'react';
+import { Heading, Button, Row } from '@datapunt/asc-ui';
+import useForm from 'react-hook-form';
 import useAppReducer from 'shared/hooks/useAppReducer';
 import { REDUCER_KEY as LOCATION } from 'shared/reducers/location';
-import ManageFormComponent from './ManageFormComponent';
+import { initalValues } from './definitions/FormFields';
+import ManageFormStyle, {
+  ControlsColumn,
+  ButtonsColumn,
+} from './ManageFormStyle';
+import FormFields from './definitions/FormFields';
+import FormInput from './components/FormInput';
 
+/**
+ *
+ * @param {string} date in dd/MM/yy format
+ *
+ * Temporary function to handle the date that comes from the server.
+ * It will be replaced when the retrieved date will be in ISO format
+ *
+ */
+const getDate = date => {
+  const regExp = /\d{2}\/\d{2}\/\d{2}/;
+
+  return date && date.match(regExp)
+    ? new Date(
+        `20${date
+          .split('/')
+          .reverse()
+          .join('-')}`
+      ).toISOString()
+    : null;
+};
+
+/**
+ *
+ * @param {object} item
+ *
+ * Converts the server feature to a client location object
+ *
+ */
 const normalize = item => {
   if (!item) return initalValues;
 
   const {
     geometry: { coordinates },
-    properties,
+    properties: {
+      description,
+      locatie_id,
+      spot_type,
+      jaar_blackspotlijst,
+      status,
+      actiehouders,
+      tasks,
+      start_uitvoering,
+      eind_uitvoering,
+      jaar_oplevering,
+      notes,
+    },
   } = item;
   return {
     ...initalValues,
-    naam: properties.description,
-    nummer: properties.locatie_id,
+    naam: description,
+    nummer: locatie_id,
     coordinaten: `${coordinates[1]}, ${coordinates[0]}`,
+    spot_type: spot_type,
+    jaar_blackspotlijst: jaar_blackspotlijst,
```

Comment:
`jaar_blackspotlijst,` should be enough, no?

Reply:
indeed! thanks!